### DISCUSSION TITLE
Version numbers in composer.json shouldn't contain letters

### DISCUSSION THREAD

**Discussion by rimace** [DirectoryTree/LdapRecord#615] [General]
Hello.
Just an inconsistency I noticed while browsing the code. Some version number constraints are containing the letter "v".
Here is a example 

    "require": {
        "psr/simple-cache": "^v1.0|^v2.0|^3.0",
        "illuminate/contracts": "^8.0|^9.0|^10.0",
        "illuminate/collections": "^8.0|^9.0|^10.0"
    },

It works, but `psr/simple-cache` has no "v" in their version numbers. That is something i also noticed in other projects of DirectoryTree. And I think it doesn't look nice.

This is more like an info for you. Maybe it was a mistake and you want to fix it (?). Anyway, thanks for reading!

Greetings,
Marcel

**Comment by stevebauman** [comment]
Fixed, thanks! 🙏
